modules/react/layout/lib/Flex.tsx

```diff
@@ -5,7 +5,7 @@ import {Box, BoxProps} from './Box';
 
 import {flex, FlexStyleProps} from './utils/flex';
 
-export type FlexProps = BoxProps & FlexStyleProps;
+export type FlexProps = Omit<BoxProps, 'display'> & FlexStyleProps;
```

Comment:
I don't see  `display` as part of the React upgrade. Why is the `Omit` needed now? Was the previous code not properly restricting the `display` prop?